components/SharedLink/SharedLink.tsx

```diff
@@ -8,14 +8,25 @@ import RelatedItems from "@/components/Shared/RelatedItems";
 import { type Work } from "@nulib/dcapi-types";
 import WorkTopInfo from "@/components/Work/TopInfo";
 import WorkViewerWrapper from "@/components/Work/ViewerWrapper";
+import { formatDateLong } from "@/lib/utils/date-helpers";
 import { getWorkSliders } from "@/lib/work-helpers";
+import { styled } from "@/stitches.config";
+
+const BoldText = styled("span", {
+  fontFamily: "$northwesternSansBold",
+});
```

Comment:
This font types are so funny. I wonder if we should create some smaller helper components for this in the future? Would `strong` make more sense semantically?

```suggestion
const BoldText = styled("strong", {
  fontFamily: "$northwesternSansBold",
  fontWeight: "400",
});
```